templates/list-action.tmpl

```diff
@@ -6,7 +6,7 @@
       {% if content.block.cancel_text %}
         {{ content.block.cancel_text }} <br>
```

Comment:
Looking at the Overflow diagram it looks like `cancel_text` is no longer used. If so, this needs to be removed from the schema. If this feature is no longer required the feature should be removed (from this template, test schemas, json schema and eq-translations).

Reply:
As discussed, I will remove them from schemas and feature will be removed on separate card. Thanks!